interface.js

```diff
@@ -0,0 +1,66 @@
+function getPostData() {
+    return fetch(`https://content.guardianapis.com/search?section=environment&api-key=0e5a597b-efce-48ec-8dd8-7e522777c0ea&show-fields=all`)
+    .then(response => {
```

Comment:
Maybe use a separate file to store the API key and use gitignore to keep it private